Remotion/ObjectBinding/Web/UI/Controls/BocValidationErrorIndicatorColumnDefinition.cs

```diff
@@ -39,5 +42,16 @@ protected override IBocColumnRenderer GetRendererInternal (IServiceLocator servi
 
     /// <inheritdoc />
     protected override string DisplayedTypeName => "ValidationErrorIndicatorColumnDefinition";
+
+    //TODO RM-8846 Not sure whether we should actually get the repo for this, but I think it's alright if we just have it sortable at any time.
+    public bool IsSortable => true;
```

Comment:
```suggestion
    public bool IsSortable => true;
```

It's normally either configurable or fixed. I would recommend making it setable so the application developer can decide if they actually want to enable/disable it. Default is enabled. See BocValueColumnDefionition for how it usually looks like, including the attributes we apply.

Reply:
Would there be any reason to not have them as auto-properties? Or should I use everything in like BocValueColumnDefinition other than that?

Reply:
The other properties are still normal ones, correct?

The practical reason for making a backing field is that you can set the default-value without needing the ctor-method. If there's multile non-chained ctors, this can be easier / less hassle. 

The stylistic reason is simply not mixing the implementations, making the code of each type more predictable. These days, there's a certain laissez-faire approach. Sometimes, it's also practical to just update the other proerties too. here, probably not?